### ISSUE TITLE
Image loading progress bar doesn't have accessible name

### ISSUE THREAD

**Issue by mcharg** [DOI-ONRR/onrr.gov-site#1974]
From OCIO Report: 

OCIO Violation: Provide alternative text for images
Occurance: 10%
Severity: 10
Priority: 10
Note: This svg element does not have a mechanism that allows an accessible name value to be calculated

Description: This is a single element with 137 instances. It’s the circular progress bar when any page is slow to load

![image.png](https://images.zenhubusercontent.com/63482c003bae1bf4351c6f26/b400aa25-6ade-4e59-b648-6385bec22317)

From the homepage Lighthouse report.
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/270952fe-5542-4e3a-80f8-4790b0361f14)

**Comment by mcharg** [comment]
Reporting Home page
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/af08efa8-f1c9-484e-bc4c-7533b9bc8859)

**Comment by mcharg** [comment]
Reporting > Geothermal
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/54509603-e9c7-4f9d-b38d-d42ceb451cef)

**Comment by mcharg** [comment]
How Valuation Works
![image.png](https://images.zenhubusercontent.com/5c589fe5e180620812ea22aa/e6e7c5bf-3963-4ed0-9cd7-68612c9f46aa)

**Comment by eelzi-ONRR** [comment]
OCIO occurance 11%, severity 10

Shows up in the OCIO report as "Provide alternative text for images"

**Comment by eelzi-ONRR** [comment]
Still showing up in the OCIO report as of June 2023

**Comment by eelzi-ONRR** [comment]
Changed the priority to High based off the OCIO priority rating

**Comment by eelzi-ONRR** [comment]
This was resolved with issue #2491
